pages/terms.js

```diff
@@ -1,21 +1,29 @@
 import Layout from '../components/layout';
-import { Hero } from '../components/jambonz-ui';
-import { getData } from '../lib/data';
+import { Hero, TextLayout } from '../components/jambonz-ui';
+import { getData, getParsedMarkdown } from '../lib/data';
 
-export default function Terms({ data }) {
+// You'll need to load the nodejs path module
+import path from 'path';
+
+export default function Terms({ data, parsed }) {
   return (
     <Layout siteData={data.site}>
       <Hero data={data.terms.hero} subStyle="terms" />
+      <TextLayout data={parsed} />
     </Layout>
   );
 }
 
 export async function getStaticProps() {
   const data = getData('terms');
+  
+  // Use the new public method in a more ad hoc manner like this...
```

Comment:
Same nit pick here, we just don't need these comments specifically. Again, same goes for `privacy.js`.